### PR TITLE
Implement react-otp-input within docz docmentation

### LINKED ISSUE #210: Implement component within docz's playground
- **Describe the Feature Request**
From issue #36 it's clear that we're moving to docz for documentation purposes. However, docz also provides functionality to directly embed the component in the documentation, along with the code (see [this](https://www.docz.site/docs/built-in-components). This is something we haven't added yet, and we should.

- **Describe Preferred Solution**
Implement OtpInput within the documentation playground.

Kindly assign this issue to me if you think its suitable. Thanks!

### PR DESCRIPTION
**Pull request by nirmalhk7**
- **What does this PR do?**
Fixed #210

- **Any background context you want to provide?**
I initially wanted to render the demo in the README.md file. If you copy-paste all content from `demo.mdx` into `README.md`, it works really well on the docz server, but that JSX is also printed directly (with no formatting) in markdown. Therefore we'd have had a great documentation website, but ugly JSX code in the repository's README. That's why I felt it'd be better if the demo is kept separately from the README.

If there's a way for JSX to be shown on docz but not on README, please let me know. I've searched online but I couldn't find any way for it.

- **Screenshots and/or Live Demo**

Screenshot from 2020-10-08 12-17-47

### CODE REVIEW

**Comment by nirmalhk7** [comment]
@apollonian Please review. Thanks!

**Comment by nirmalhk7** [comment]
Please let me know if there are any changes to be done, thank you!

File: README.md [hunk 1]
```diff
@@ -200,25 +200,3 @@ Feel free to open [issues](https://github.com/devfolioco/react-otp-input/issues/
 ## License
 
 [![NPM](https://img.shields.io/npm/l/react-otp-input)](https://github.com/devfolioco/react-otp-input/blob/master/LICENSE)
-
-## Contributors ✨
```
Comment (apollonian) [inline README.md:204]: Does docz error out when we have this section in the README?

**Comment by prateek3255** [comment]
Thanks for working on this @nirmalhk7 and sorry for the delayed response. We updated the file structure with v3 can you reopen the PR with the latest changes?

**Pull request closed without merging**